cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -20,9 +52,40 @@ def test_ec2_targeted_refresh_instance():
             2. Instance RUNNING
             3. Instance STOPPED
             4. Instance UPDATE
-            5. Instance DELETE - or - Instance TERMINATE
+            5. Instance RUNNING
+            6. Instance DELETE - or - Instance TERMINATE
     """
-    pass
+    vms_collection = appliance.rest_api.collections.vms
+    flavors_collection = appliance.rest_api.collections.flavors
+
+    # create
+    template_id = provider.mgmt.get_template(
+        provider.data.templates.get('small_template').name).uuid
+    instance = provider.mgmt.create_vm(template_id, vm_name=random_vm_name('refr'))
```

Comment:
I think this should use the `create_vm` fixture. https://github.com/ManageIQ/integration_tests/blob/a2b8ad55d1057be6f517759a80b350b335e53de7/cfme/fixtures/vm.py#L131

Reply:
Actually I wanted to do it all via REST API. As I can see create_vm utilizes UI. Does it make sense to use UI for creating vm and for the rest to use REST API?

Reply:
`create_vm` will actually use `create_on_provider` which will then use `wrapanapi` to deploy the template for the vm. https://github.com/ManageIQ/integration_tests/blob/a2b8ad55d1057be6f517759a80b350b335e53de7/cfme/fixtures/vm.py#L46 

So it shouldn't be using the UI with `create_vm`

Reply:
Okay, so it just probably checks whether vm is available, because it opens browser at vm details.

Reply:
Changed to use create_vm.